web/Views/Shared/Details/_PropertyAccepts.cshtml

```diff
@@ -1,58 +1,58 @@
-                <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsWithoutGuarantor)"></i>
-                Without rent guarantor
-            </div>
-        }
-    </div>
+﻿@model PropertyDetailsViewModel
+
+@functions
+{
+    string GetPreferenceFont(bool? preference)
+    {
+        return preference == true ? "" : "font-blue";
+    }
+
+    string GetPreferenceIcon(bool? preference)
+    {
+        return preference == true ? "bi-check-lg" : "bi-x-lg";
+    }
+}
+
+<div class="card custom-card flex-grow-1 mt-2">
+    <div class="card-body">
+        <div class="@GetPreferenceFont(Model.Property!.LandlordRequirements.AcceptsSingleTenant)">
+            <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsSingleTenant)"></i>
+            Single tenants
+        </div>
+        <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsCouple)">
+            <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsCouple)"></i>
+            Couples
+        </div>
+        <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsFamily)">
+            <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsFamily)"></i>
+            Families
+        </div>
+        <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsPets)">
+            <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsPets)"></i>
+            With pets
+        </div>
+        @if (User.IsLandlordOrAdmin())
+        {
+            <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsUnder35)">
+                <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsUnder35)"></i>
+                Is under 35
+            </div>
+            <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsCredit)">
+                <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsCredit)"></i>
+                On universal credit
+            </div>
+            <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsBenefits)">
+                <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsBenefits)"></i>
+                On housing benefits
+            </div>
+            <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsNotInEET)">
+                <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsNotInEET)"></i>
+                Not in EET (Employment, Education, or Training)
+            </div>
+            <div class="@GetPreferenceFont(Model.Property.LandlordRequirements.AcceptsWithoutGuarantor)">
+                <i class="bi me-1 @GetPreferenceIcon(Model.Property.LandlordRequirements.AcceptsWithoutGuarantor)"></i>
+                Accepts without guarantor
```

Comment:
I think this should just say 'without guarantor', because these all come below the 'Property accepts' heading, and it would read weirdly if 'accepts' was there twice :)